File: code_combat/7_Sarven_Desert/274-Medical_Attention/medical_attention.py
while True:
    currentHealth = hero.health
    if currentHealth < hero.maxHealth / 2:
        hero.moveXY(65, 46)

    enemy = hero.findNearestEnemy()
    if enemy:
        distance = hero.distanceTo(enemy)
        if currentHealth < hero.maxHealth and hero.canCast("drain-life") && distance <= 15:
            hero.cast("drain-life", enemy)
        elif hero.canCast("chain-lightning") and distance <= 30:
            hero.cast("chain-lightning", enemy)
        else:
            hero.attack(enemy)

File: projects/code_combat/7_Sarven_Desert/274-Medical_Attention/medicalAttention.js
// Ask the healer for help when you're under one-half health.

while (true) {
    var currentHealth = hero.health;
    var healingThreshold = hero.maxHealth / 2;
    // If your current health is less than the threshold,
    // move to the healing point and say, "heal me".
    // Otherwise, attack. You'll need to fight hard!
    if (currentHealth < healingThreshold) {
        hero.moveXY(65, 46);
    }

    var enemy = hero.findNearestEnemy();
    if (enemy)
        hero.attack(enemy);
}
